src/Arcus.WebApi.Telemetry.Serilog/Correlation/CorrelationInfoEnricher.cs

```diff
@@ -35,7 +36,8 @@ public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
             Guard.NotNull(logEvent, nameof(logEvent));
             Guard.NotNull(propertyFactory, nameof(propertyFactory));
 
-            var correlationInfo = _services.GetRequiredService<HttpCorrelationInfo>();
+            var correlationInfoAccessor = _services.GetRequiredService<ICorrelationInfoAccessor>();
```

Comment:
We're in web API here, I think we should use `HttpCorrelationInfoAccessor` since that is what we are registering, no?

However, we might not care about it here since both return `CorrelationInfo` 🤷‍♂

Reply:
Yes, indeed, it doesn't matter here bc they both return the same and are registered in such a way that (when you use the default correlation model) you can use both.